package.json

```diff
@@ -7,10 +7,13 @@
     "ejs": "^2.4.1",
     "express": "^4.10.1",
     "express-session": "^1.13.0",
+    "fs-extra": "^0.28.0",
```

Comment:
Is fs-extra, multiparty, or ng-file-upload all used on the front end only, or are these backend as well?